cds/modules/records/tasks.py

```diff
@@ -267,91 +273,128 @@ def file_integrity_report():
 def subformats_integrity_report(start_date=None, end_date=None):
     """Send a report of all corrupted subformats to CDS admins."""
     report = []
+    update_cache = True
```

Comment:
HI, thanks for the feedback, I used another solution because we needed to check the file accessibility for master and subformats as well. Please take a look when you have the time.